install-usercss/install-usercss.js

```diff
@@ -1,46 +1,25 @@
 /* global CodeMirror semverCompare closeCurrentTab messageBox download
-  $ $$ $create $createLink t prefs API getTab */
+  $ $$ $create $createLink t prefs API */
 'use strict';
 
 (() => {
-  const DUMMY_URL = 'foo:';
-
   // TODO: remove .replace(/^\?/, '') when minimum_chrome_version >= 52 (https://crbug.com/601425)
   const params = new URLSearchParams(location.search.replace(/^\?/, ''));
-  let liveReload = false;
+  const tabId = params.has('tabId') ? Number(params.get('tabId')) : -1;
+  const initialUrl = params.get('updateUrl');
+  if (!initialUrl) throw 'No updateUrl parameter';
```

Comment:
Definitely not. It doesn't make any difference since this is a fatal exception that no one will catch. I'd rather remove this entire line because if some user tried to mess around they should accept the consequences of things totally breaking.